opencensus/trace/tracers/context_tracer.py

```diff
@@ -116,6 +117,8 @@ def end_span(self, *args, **kwargs):
             execution_context.set_current_span(cur_span.parent_span)
         else:
             execution_context.set_current_span(None)
+            self.span_context.trace_id = generate_trace_id()
+            self.trace_id = self.span_context.trace_id
```

Comment:
`tracer.trace_id` is unused, and I don't know why it's included on the tracer. Removing it is a problem for another PR.